tests/system/python/e2e/test_e2e_modbus_c_pi.py

```diff
@@ -0,0 +1,167 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Test system/python/e2e/test_e2e_modbus_c_pi.py
+
+"""
+
+import http.client
+import json
+import time
+import pytest
+import socket
+import utils
+
+
+__author__ = "Vaibhav Singhal"
+__copyright__ = "Copyright (c) 2019 Dianomic Systems"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+SOUTH_PLUGIN = "modbus-c"
+PLUGIN_NAME = "ModbusC"
+SVC_NAME = "modbus-c"
+SOUTH_PLUGIN_LANGUAGE = "cpp"
```

Comment:
We are keeping it C everywhere else... plugin_lang='C'

you can avoid it a module level variable and can write it in def call directly. As test is specific